docs/data-analysis.md

```diff
@@ -1,164 +1,106 @@
 # Analyzing bot data
 
-After performing backtests, or after running the bot for some time, it will be interesting to analyze the results your bot generated.
+You can analyze the results of backtests and trading history easily using Jupyter notebooks. A sample notebook is located at `user_data/notebooks/analysis_example.ipynb`. For usage instructions, see [jupyter.org](https://jupyter.org/documentation).
```

Comment:
I think we should mention here to start the notebook-server from within the virtual environment, just to hop in front of questions that may arrise

Reply:
done